src/js/events/ChronosActions.js

```diff
@@ -133,6 +134,26 @@ const ChronosActions = {
     });
   },
 
+  updateJob: function (jobID, data) {
```

Comment:
I am not familiar with the API yet, so forgive me my question.
Is _update_ the right wording here?
At the MarathonActions we call this part `editService` for example instead of update.
Just a question?!